tests/functional/check_overstays_test.go

```diff
@@ -0,0 +1,95 @@
+package functional
+
+import (
+	"context"
+	"net/http"
+	"testing"
+	"time"
+
+	"github.com/IgorSteps/easypark/internal/adapters/rest/models"
+	"github.com/IgorSteps/easypark/internal/domain/entities"
+	"github.com/IgorSteps/easypark/tests/functional/client"
+	"github.com/IgorSteps/easypark/tests/functional/utils"
+	"github.com/stretchr/testify/suite"
+)
+
+type CheckOverStaysTestSuite struct {
+	client.RestClientSuite
+}
+
+func (s *CheckOverStaysTestSuite) TestCheckOverStays() {
+	// --------
+	// ASSEMBLE
+	// --------
+	ctx, cancel := context.WithTimeout(context.Background(), 30*time.Second)
+	defer cancel()
+
+	// Create and login an admin
+	adminToken := utils.CreateAndLoginAdmin(ctx, &s.RestClientSuite)
+
+	// Create and login a driver
+	driver, driverToken := utils.CreateAndLoginDriver(ctx, &s.RestClientSuite, nil)
+
+	// Create a parking lot with a capacity of 5
+	createParkingLot := &models.CreateParkingLotRequest{
+		Name:     "test-lot",
+		Capacity: 5,
+	}
+	parkingLot := utils.CreateParkingLot(ctx, adminToken, createParkingLot, &s.RestClientSuite)
+	parkingSpaceID := parkingLot.ParkingSpaces[0].ID
+
+	// Create a parking request that will end in 3 seconds
+	createParkingRequest := &models.CreateParkingRequestRequest{
+		DestinationParkingLotID: parkingLot.ID,
+		StartTime:               time.Now(),
+		EndTime:                 time.Now().Add(3 * time.Second),
+	}
+	parkingRequest := utils.CreateParkingRequest(ctx, driverToken, driver.ID, parkingLot.ID, createParkingRequest, &s.RestClientSuite)
+	utils.AssignParkingSpace(ctx, parkingSpaceID, parkingRequest.ID, adminToken, &s.RestClientSuite)
+
+	// Create another parking request that will end in 2 hours
+	createParkingRequest2 := &models.CreateParkingRequestRequest{
+		DestinationParkingLotID: parkingLot.ID,
+		StartTime:               time.Now(),
```

Comment:
This is failing: start time is already in the past when it reaches the backend. You should a couple of seconds to it

<img width="1369" alt="Screenshot 2024-05-15 at 14 13 42" src="https://github.com/IgorSteps/easypark/assets/84919282/7021ecf7-602e-4bb6-9eba-bb3ec3aee3bd">